airflow-core/src/airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -770,12 +770,19 @@ def post_clear_task_instances(
         )
 
     if not dry_run:
-        clear_task_instances(
-            task_instances,
-            session,
-            DagRunState.QUEUED if reset_dag_runs else False,
-            run_on_latest_version=body.run_on_latest_version,
-        )
+        if body.is_running_message:
+            for ti in task_instances:
+                ti.is_running_message = body.is_running_message
+        
+        try:
+            clear_task_instances(
+                task_instances,
+                session,
+                DagRunState.QUEUED if reset_dag_runs else False,
+                run_on_latest_version=body.run_on_latest_version,
+            )
+        except ValueError as e:
```

Comment:
did you encountered an unhandled ValueError? Those should crash the server if they are unexpected. (only throw 400 if this is actually client's falt)